File: examples-sdk/go/702-mq-sqs-send-receive/main.go
// 702-mq-sqs-send-receive — Go SDK equivalent
// Compile: vil compile --from go --input 702-mq-sqs-send-receive/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}

File: examples-sdk/zig/702-mq-sqs-send-receive/main.zig
// 702-mq-sqs-send-receive — Zig SDK equivalent
// Compile: vil compile --from zig --input 702-mq-sqs-send-receive/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}
